app/src/main/java/es/wolfi/app/passman/VaultViewAdapter.java

```diff
@@ -88,6 +91,33 @@ public void onClick(View v) {
         });
     }
 
+    private CharSequence calculateVaultState(int position) {
+        boolean unlocked = false;
+        boolean active = false;
+
+        // Active vault
+        final Vault av = Vault.getActiveVault();
+
+        // Current vault
+        Vault v = mValues.get(position);
+
+        if (v != null) {
+            active = (av != null && v.guid.equals(av.guid));
+            unlocked = v.is_unlocked();
+        }
+
+        if (unlocked && active)
+            return "Active/Unlocked";
```

Comment:
App is localized to several languages, this strings should be localisable too as they seem to be shown to the user